editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -502,6 +464,9 @@ void CanvasItemEditor::_find_canvas_items_at_pos(const Point2 &p_pos, Node *p_no
 			CanvasLayer *cl = p_node->cast_to<CanvasLayer>();
 			_find_canvas_items_at_pos(p_pos, p_node->get_child(i), transform, cl ? cl->get_transform() : p_canvas_xform, r_items); //use base transform
 		}
+
+		if (limit && r_items.size() >= limit)
```

Comment:
Well, this is common in C / C++ ... And I personnaly find this more readable than with > 0.
If some of the lead developpers asks for it i'll change it, but I dont think its necessary.